Altis_Life.Altis/core/items/fn_spikeStrip.sqf

```diff
@@ -6,28 +6,22 @@
     Description:
     Creates a spike strip and preps it.
 */
-private ["_spikeStrip"];
-if (!isNil "life_action_spikeStripPickup") exitWith {hint localize "STR_ISTR_SpikesDeployment"};    // avoid conflicts with addactions allowing duplication.
-_spikeStrip = "Land_Razorwire_F" createVehicle [0,0,0];
+private _spikeStrip = "Land_Razorwire_F" createVehicle [0,0,0];
 _spikeStrip attachTo[player,[0,5.5,0]];
 _spikeStrip setDir 90;
-_spikeStrip setVariable ["item","spikeDeployed",true];
 
-life_action_spikeStripDeploy = player addAction[localize "STR_ISTR_Spike_Place",{if (!isNull life_spikestrip) then {detach life_spikeStrip; life_spikeStrip = objNull;}; player removeAction life_action_spikeStripDeploy; life_action_spikeStripDeploy = nil;},"",999,false,false,"",'!isNull life_spikestrip'];
+life_action_spikeStripDeploy = player addAction[localize "STR_ISTR_Spike_Place",{detach life_spikeStrip; life_spikeStrip = objNull; player removeAction life_action_spikeStripDeploy; life_action_spikeStripDeploy = nil;},"",999,false,false,"",'!isNull life_spikestrip'];
 life_spikestrip = _spikeStrip;
-waitUntil {isNull life_spikeStrip};
```

Comment:
This waitUntil is needed to halt the script for the current functionality, aswell as removing the action. Another method would be to make a separate function for when the spikes are placed, running the remaining code then.